okhttp/src/main/java/okhttp3/CipherSuite.java

```diff
@@ -395,8 +395,7 @@
   // public static final CipherSuite TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256 = of("TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256", 0xccad);
   // public static final CipherSuite TLS_RSA_PSK_WITH_CHACHA20_POLY1305_SHA256 = of("TLS_RSA_PSK_WITH_CHACHA20_POLY1305_SHA256", 0xccae);
 
-  // TLS 1.3 https://tools.ietf.org/html/draft-ietf-tls-tls13-28
-  // Documentation only ids will be updated once assigned by IANA.
+  // TLS 1.3 https://tools.ietf.org/html/rfc8446
```

Comment:
Until now I hadn’t even realized that these don’t overlap with the other ones. That’s actually going to be very handy!

Reply:
Awkwardly, Conscrypt will ignore and only use the first three for now, no matter what you ask for.